samples/MvvmSample.Core/ViewModels/AsyncRelayCommandPageViewModel.cs

```diff
@@ -4,12 +4,13 @@
 
 using System.Threading.Tasks;
 using CommunityToolkit.Mvvm.Input;
+using MvvmSample.Core.Services;
 
 namespace MvvmSample.Core.ViewModels;
 
 public class AsyncRelayCommandPageViewModel : SamplePageViewModel
 {
-    public AsyncRelayCommandPageViewModel()
+    public AsyncRelayCommandPageViewModel(IFilesService filesService) : base(filesService)
```

Comment:
Nit: all these `base` calls should be in a new line.
```suggestion
    public AsyncRelayCommandPageViewModel(IFilesService filesService)
        : base(filesService)
```

Reply:
fixed